### PR TITLE
PRSD-365: Creates Landlord Registration Start Page

### PR DESCRIPTION
**Pull request by isobel-softwire**
- Adds `/register-as-a-landlord` GET endpoint in `RegisterLandlordController`.
- Permits all user requests matching `/register-as-a-landlord`
- Changes `button` fragment to `startButton`
- Stops Thymeleaf from overwriting `startButton`'s icon
- Adds details about the project's use of the Gov.UK design system to the `ReadMe`
- Adds `registerAsALandlord` template (content provided via `messages.properties`)
- Adds tests for the register a landlord controller and page

UI: 
image

### CODE REVIEW

**Comment by Travis-Softwire** [comment]
For UI changes, can we get a screenshot or videoclip attached to the PR as well?

**Comment by Travis-Softwire** [comment]
I've left a few comments, but overall I think this is looking good

File: src/main/resources/templates/registerAsALandlord.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html th:replace="~{fragments/layout :: layout(${title}, ${serviceName}, ~{::main})}">
+<main class="govuk-main-wrapper" id="main-content">
+    <header>
+        <h1 class="govuk-heading-xl" th:text="#{registerAsALandlord.heading}">Private Rented Sector Database</h1>
```
[the remark this answers is not in the document]
Comment (isobel-softwire) [reply src/main/resources/templates/registerAsALandlord.html:5]: I agree that leaving the real text is misleading, but doesn't the th:text property already let the person editing `messages.properties` know where the messages are going? Is it worth having message sources as both in the th:text property and the placeholder text?

[the remark this answers is not in the document]
Comment (Travis-Softwire) [reply src/main/resources/templates/registerAsALandlord.html]: So I think the advantage is that someone can see roughly where on the page different bits of text go by opening the template html file in a browser without having to know html. I agree it doesn't add anything if the person knows how to read the html, but I think at this point we don't know if that will be the case.

[the remark this answers is not in the document]
Comment (isobel-softwire) [reply src/main/resources/templates/registerAsALandlord.html]: Ok, that makes sense

**Review by Travis-Softwire — APPROVED** [review]
LGTM :) 

Before merging you might want to rebase this onto main once you've merged the playwright ticket and remove those commits here - shout if you need a hand with that

**Pull request merged**